### ISSUE TITLE
Ditch the "community" in "community translations"

### ISSUE THREAD

**Issue by jgroenen** [standard-for-public-code/community-translations-standard#42]
Since the standard itself is now a community project, maybe ditch the "community" in "community translations"?

**Comment by ElenaFdR** [comment]
Yup! Is it worth stressing that none of the translations are automatically machine generated, or is that self evident by the invitation for people to start their own translation?

**Comment by jgroenen** [comment]
Would it be OK to bring the codefornl/community-translations-standard back here, but not in this repo, but as

"standard-for-public-code/standard-for-public-code-translation-nl"

We can do the same for "es" and "tw" and "dk", and then make this one into

"standard-for-public-code/standard-for-public-code-translation-template"?

**Comment by ElenaFdR-ICATT** [comment]
The repo About summary currently says:
> Unofficial translations of the Standard for Public Code, provided by the community.

Suggested new wording:
> Courtesy translations of the Standard for Public Code.

There might be marketing argument for leaving "provided by the community." in the summary, as this also appears when you search in Github.

<img width="1250" height="404" alt="Image" src="https://github.com/user-attachments/assets/b03fc62f-edf6-4c4b-8f05-6ea632bcc73c" />

NB: This change can only be made by someone with admin rights to this repo.
